scheduler/kubernetes/snaps.md

```diff
@@ -11,218 +11,111 @@ meta-description: "Learn to take a snapshot of a volume from a Kubernetes persis
 * TOC
 {:toc}
 
-This document will show you how to create snapshots of Portworx volumes and use them in pods.  It uses MySQL as an example.
+This document will show you how to create snapshots of Portworx volumes and how you can clone those snapshots to use them in pods.
 
 ## Managing snapshots with `kubectl`
 
-### Taking snapshots
-
-Following are the different ways in which you can take snapshots of your volume through kubernetes.
-
-#### Periodic snapshots
-When you create a StorageClass, you can specify a snapshot schedule on the volume as specified below. This allows to snapshot the persistent data of the running pod using the volume.
-```yaml
-kind: StorageClass
-apiVersion: storage.k8s.io/v1beta1
-  metadata:
-    name: portworx-repl-1-snap-internal
-provisioner: kubernetes.io/portworx-volume
-parameters:
-  repl: "1"
-  snap_interval: "240"
-```
-Above spec will take snapshots of the _portworx-repl-1-snap-internal_ PVC every 240 minutes.
-
-#### On demand Snapshots
+### Pre-requisites
+This requires that you already have [STORK](/scheduler/kubernetes/stork.html) installed and running on your
+Kubernetes cluster
 
-You can trigger a new snapshot on a running POD by creating a PersistentVolumeClaim
-
-##### Using annotations
+### Taking snapshots
 
-Portworx uses a special annotation `px/snapshot-source-pvc` which can be used to identify the name of the source PVC whose snapshot needs to be taken.
+If you have a PVC called mysql-data backed by a Portworx volume, you can create a snapshot for that PVC by
```

Comment:
We cannot remove the old content as people are already using it and they need access to the steps they used. Rather keep the new method as the default on this page. At the bottom, have a section that says "If you need to instructions to use the previous snapshots methods using annotations or inline specs, refer [this page](here comes the url with old content)

Reply:
Added a NOTE at the top. If people are looking for it'll make it easier for them to find it.